sdformat_to_mjcf/tests/test_add_link.py

```diff
@@ -61,81 +59,27 @@ def test_link_inertia(self):
         # <link name="base_link">
         #   <inertial>
         #       <pose degrees="true">1 2 3   90 60 45</pose>
-        #       <mass>384</mass>
+        #       <mass>10</mass>
```

Comment:
what is the reason for the changes in this file starting with this line? it reverts changes from https://github.com/gazebosim/gz-mujoco/pull/5